docs/src/content/docs/fr/guides/sidebar.mdx

```diff
@@ -208,8 +203,7 @@ title: Ma page
 sidebar:
   # Définit une étiquette personnalisée pour le lien dans la barre latérale
   label: Étiquette personnalisée
-  # Définit un ordre personnalisé pour le lien (les nombres plus petits
-  # sont affichés plus haut)
+  # Définit un ordre personnalisé pour le lien (les nombres plus petits sont affichés plus haut)
```

Comment:
This comment now requires scrolling to see the entire comment on desktop compared to the previous version. I think this should be reverted.